src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -221,34 +221,77 @@ export interface TransitionDirectionalAnimations {
 }
 ```
 
-The following example shows all the necessary properties to define a custom `fade` animation:
+The following example shows all the necessary properties to define a custom transition animation.
+
+In this instance, the custom animation `crash` is defined inside a `<style is:global>` tag in your root layout file:
+
+```astro
+---
+import ViewTransitions from "astro/components/ViewTransitions.astro";
+---
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+    <meta name="viewport" content="width=device-width" />
+    <meta name="generator" content={Astro.generator} />
+    <title>Astro</title>
+    <ViewTransitions />
+  </head>
+  <body>
+    <slot />
+  </body>
+</html>
+
+<style is:global>
+  @keyframes crash {
+    0% {
+      opacity: 0;
+      transform: scale(1) translateX(200px);
+    }
+    50% {
+      opacity: 0.5;
+      transform: scale(1.1);
+    }
+    100% {
+      opacity: 1;
+      transform: scale(1) translateX(0);
+    }
+  }
+</style>
+
+```
+
+Remember to define the animation behavior in the frontmatter of all the components using the animation:
 
 ```astro
 ---
 const anim = {
   old: {
-    name: 'fadeIn',
-    duration: '0.2s',
-    easing: 'linear',
-    fillMode: 'forwards',
+    name: "crash",
+    duration: "0.5s",
+    easing: "ease-in",
+    fillMode: "forwards",
   },
   new: {
-    name: 'fadeOut',
-    duration: '0.3s',
-    easing: 'linear',
-    fillMode: 'backwards',
-  }
+    name: "crash",
+    duration: "0.5s",
+    easing: "ease-in-out",
+    fillMode: "backwards",
+  },
 };
 
-const myFade = {
-	forwards: anim,
-	backwards: anim,
+const customTransition = {
+  forwards: anim,
+  backwards: anim,
 };
 ---
 
-<header transition:animate={myFade}> ... </header>
+<header transition:animate={customTransition}> ... </header>
 ```
 
+
```

Comment:
Maybe we should add a sentence saying that you can of course use different objects for forward and backward and different keframe names for old and new?